www/m/index.js

```diff
@@ -1091,6 +1090,34 @@ function updateInfo() {
 	});
 }
 
+function updateEntropy() {
+	r2.cmd ("p=", function(d) {
```

Comment:
I'm now using p=ej with 50 blocks (the chart width is 500px so that's why).
I've also changed double quotes to single ones.